content/src/content/jcr_root/apps/core/wcm/sandbox/components/carousel/v1/carousel/carousel.html

```diff
@@ -23,7 +23,7 @@
         <div data-sly-repeat.item="${carousel.items}"
              data-sly-resource="${item.path @ decorationTagName='div'}"
              role="tabpanel"
-             class="cmp-carousel__item"
+             class="cmp-carousel__item${itemList.first ? ' cmp-carousel__item--active' : ''}"
```

Comment:
Shouldn't it be `item` instead of `itemList`? (Same below).

Reply:
first is a property of the `itemList` object not the `item`

Reply:
ah yes, HTL specific object!